zcash_primitives/src/transaction/components/amount.rs

```diff
@@ -32,6 +38,14 @@ impl Amount {
         Amount(0)
     }
 
+    /// Creates a constant Amount from an i64.
+    ///
+    /// Panics: if the amount is outside the range `{-MAX_MONEY..MAX_MONEY}`.
+    pub const fn const_from_i64(amount: i64) -> Self {
+        assert!(-MAX_MONEY <= amount && amount <= MAX_MONEY); // contains is not const
+        Amount(amount)
+    }
```

Comment:
Despite the codecov warning, this is in fact covered by tests. (A test uses `MINIMUM_FEE` which uses `const_from_i64`.)